### PR TITLE
assign activities and assign diagnostics updates

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Visual updates to the Assign Activities and Assign Diagnostics pages.

## WHY
We wanted to spruce up these pages and make them more consistent with the design language we're using going forward.

## HOW
React and CSS, and updating existing parts of these pages to use Backpack components.

### Screenshots
Screenshot 2023-09-27 at 11 06 28 AM
Screenshot 2023-09-27 at 11 06 11 AM

### Notion Card Links
https://www.notion.so/quill/UI-Updates-to-the-Assign-Activities-Page-Front-end-P2-02d4ea0c8935469e9bf4601b17b42e77?pvs=4

https://www.notion.so/quill/UI-Fix-Assign-Activities-Diagnostics-Front-End-P4-0e06cf23e71d4affa58ab3a72680dba8?pvs=4

https://www.notion.so/quill/Assign-Activity-Iteration-on-Evidence-Activity-Feed-Front-End-P5-41361e96d713427698631a46498e07af?pvs=4

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | YES

### CODE REVIEW

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx [hunk 1]
```diff
@@ -111,48 +143,74 @@ const AssignANewActivity = ({ numberOfActivitiesAssigned, showDiagnosticBanner }
...
-            })}
-          </div>
-        </tbody>
-      </table>
-    </div>
-  )
+      <DataTable
+        headers={headers}
+        rows={rows}
+      />
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx:210]: I appreciate this refactor to use the `DataTable` component

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx [hunk 2]
```diff
@@ -111,48 +143,74 @@ const AssignANewActivity = ({ numberOfActivitiesAssigned, showDiagnosticBanner }
...
+      {showAllSuggestedActivities ? null : <button className="quill-button outlined small secondary focus-on-light show-all-button" onClick={handleClickShowAll} type="button">Show all {activitiesToSuggest.length} Reading for Evidence activities</button>}
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx:211]: Does this need an `aria-label`?

**Review by eadams17 — APPROVED** [review]
This looks great, thanks for updating!

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx (hunk 2 above)
Comment (emilia-friedberg) [reply services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx:211]: @eadams17 no, the text of the button functions as a label. `aria-label` is only necessary when the button doesn't contain anything that explains what it's doing (like if it's just an image, and the alt text. doesn't clarify the purpose of the button).

**Pull request merged**